docs/janssen-server/to-be-placed/google.md

```
@@ -25,7 +25,7 @@ The following steps explain how to create credentials for your project. Your app
 ## Configure jans-auth server
 
 ### Download Google Client JSON file
-On the following page, https://console.cloud.google.com/apis/credentials, you will see a table containing your recently created client. Click on the download button and download the JSON file containing details of the Client. 
+On the following page, https://console.cloud.google.com/apis/credentials, you will see a table containing your recently created client. Click on the download button and download the JSON file containing details of the Client.
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)